.github/workflows/stale.yaml

```diff
@@ -0,0 +1,56 @@
+# This workflow alerts and then closes the stale issues/PRs after specific time
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+
+name: 'Close stale issues and PR'
+on:
+  schedule:
+    - cron: "30 1 * * *"
+    
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          # comment on issue if not active for more then 14 days.
+          stale-issue-message: 'This issue has been automatically marked as stale because it has no recent activity. It will be closed if no further activity occurs. Thank you.'
+        
+          # comment on PR if not active for more then 14 days.
+          stale-pr-message: 'This PR has been automatically marked as stale because it has no recent activity. It will be closed if no further activity occurs. Thank you.'
+        
+          #comment on issue if stale for more then 14 days. 
+          close-issue-message: "Closing as stale. Please reopen if you'd like to work on this further"
+        
+          #comment on PR if stale for more then 14 days. 
+          close-pr-message: "Closing as stale. Please reopen if you'd like to work on this further"
```

Comment:
This PR was closed due to lack of activity after being marked stale for past 28 days.